abe/app.py

```diff
@@ -36,6 +51,8 @@ def default(self, obj):
 app.json_encoder = CustomJSONEncoder
 
 # add return representations
+
```

Comment:
autopep added these newlines, to bring the file into conformance with PEP8. I considered whether to move `add return representations` inside the function if it's annotating the function implementation or a line inside it, or make it a docstring if it's documenting the function behavior, but I actually can't figure out what the comment is talking about.